src/utils/recordedit-utils.ts

```diff
@@ -507,9 +567,10 @@ export function populateEditInitialValues(
  */
 export function populateSubmissionRow(reference: any, formNumber: number, formData: any, initialValues?: any[]) {
   const submissionRow: any = {};
-  const setSubmission = (col: any, skipEmpty?: boolean) => {
+  const setSubmission = (col: any, skipEmpty?: boolean, includeDisabled?: boolean) => {
     let v = formData[formNumber + '-' + col.name];
 
+    // TODO col.isDisabled is wrong. it's always returning false
```

Comment:
Is this still an issue?

Reply:
Yeah, `col` in this case is an ermrestjs `ReferenceColumn` object and `isDisabled` is not available on this object. The `populateSubmissionRow` function in `recordCreate.js`is also doing the similar check. Instead of just removing `col.isDisabled` I left this TODO so I can think about it and change it later outside of this PR.